### ISSUE TITLE
microphone PermissionStatus.unknown

### ISSUE THREAD

**Issue by haslinger** [BaseflowIT/flutter-permission-handler#138]
I am trying to record audio, so in ```app/src/profile/AndroidManifest.xml``` I added
```<uses-permission android:name="android.permission.RECORD_AUDIO"/>```
and in the actual workflow I request and check permission with: 
```
await PermissionHandler().requestPermissions([PermissionGroup.microphone]);
PermissionStatus microphone = await PermissionHandler().checkPermissionStatus(PermissionGroup.microphone);
print("microphone ${microphone}");
```

While the same procedure works perfectly fine for ```storage```, here  I get 
```
D/permissions_handler( 8148): No permissions found in manifest for: 7
D/permissions_handler( 8148): No permissions found in manifest for: 7
I/flutter ( 8148): microphone PermissionStatus.unknown
```

I saw several similar issues here in the repo, so I tried several approaches:
* Made sure, I am using 3.2.1
* Did a flutter clean
* Tried on a Samsung Galaxy Tab A with Android 8.1 and on a second device: Nokia 5 with Android 9
* Checked for a blocked permission on the device itself.
It seems to me, that the app does not know about the permission, although it is in the Manifest.

Any clues what I am doing wrong here would be highly appreciated.
The app itself is open source and the full current source code is [over there](https://github.com/informatom/recomendo).

**Comment by mvanbeusekom** [comment]
Hello @haslinger, thank you for raising this issue. I have tried our example app that is part of the plugin (see [here](https://github.com/BaseflowIT/flutter-permission-handler/tree/develop/example)) which works as intended.

Next I looked at your repo and noticed that you only included the permissions in de "profile" version of the `AndroidManifest.xml` (https://github.com/informatom/recomendo/blob/master/android/app/src/profile/AndroidManifest.xml). 

And not the "main" version. This means these settings are only applied when you are running the app in profiling mode (not in release or debug). Are you sure you are running the app using the profile scheme when you are testing this?
